node/src/actors/chain_manager/actor.rs

```diff
@@ -79,8 +78,14 @@ impl ChainManager {
                 // Do not start the MiningManager if the configuration disables it
                 act.mining_enabled = config.mining.enabled;
 
-                // Flag to indicate if we want to split the mint transaction
-                act.split_mint = config.mining.split_mint;
+                // External address mint sharing parameters
+                act.external_address = if let Some(s) = &config.mining.mint_pkh {
+                    PublicKeyHash::from_str(s.as_str()).ok()
+                } else {
+                    None
+                };
```

Comment:
I'd prefer not to destructure the `Option` if the whole expression is returning a new `Option`, we make the most of `and_then` for that:
```suggestion
                act.external_address = &config
                    .mining.mint_pkh.and_then(|pkh|
                        PublicKeyHash::from_str(pkh.as_str()).ok()
                    };
```